GovUkDesignSystemComponents/TableRowViewModel.cs

```diff
@@ -0,0 +1,49 @@
+﻿using GovUkDesignSystem.GovUkDesignSystemComponents.SubComponents;
+using System;
+using System.Collections.Generic;
+
+namespace GovUkDesignSystem.GovUkDesignSystemComponents
+{
+    public class TableCellViewModel : IHtmlText
+    {
+        /// <summary>
+        ///     If html is set, this is not required. Text for cells in table rows. If html is provided, the text argument will be ignored.
+        /// </summary>
+        public string Text { get; set; }
+
+        /// <summary>
+        ///      If text is set, this is not required. HTML for cells in table rows. If html is provided, the text argument will be ignore.
+        ///     <br/>Required. 
+        /// </summary>
+        public Func<object, object> Html { get; set; }
+
+        /// <summary>
+        ///     Specify format of a cell. Currently we only use "numeric".
+        /// </summary>
+        public string Format { get; set; }
+
+        /// <summary>
+        ///     Classes to add to the table row cell. 
+        /// </summary>
+        public string Classes { get; set; }
+
+        /// <summary>
+        ///     Specify how many columns a cell extends.
+        /// </summary>
+        public int Colspan { get; set; } = -1;
```

Comment:
Might be nicer to have these as nullable instead of using magic values

Reply:
Yes, certainly. Done, thanks